packages/video_player/video_player_web/example/integration_test/video_player_web_test.dart

```diff
@@ -181,26 +183,60 @@ void main() {
         },
       ).toList();
 
+      await VideoPlayerPlatform.instance.setVolume(videoPlayerId, 0);
+      await VideoPlayerPlatform.instance.play(videoPlayerId);
+      await VideoPlayerPlatform.instance.play(videoPlayerId);
+
+      // Let the video play, until we stop seeing events for two seconds
```

Comment:
The timeout of this stream isn't modified (line 180), so this is still waiting for 1 second.